lib/Reader/reader.cpp

```diff
@@ -6564,6 +6569,7 @@ void ReaderBase::readBytesForFlowGraphNodeHelper(
       goto GEN_BRANCH;
 
     GEN_BRANCH:
+      nop();
```

Comment:
FWIW, it might be nice to change the name of this function to `debug_nop` or similar to indicate that nothing is emitted unless we're compiling for debug.